src/modules/aws_account@0.0.1/migration/1644463885917-aws_account.ts

```diff
@@ -5,9 +5,31 @@ export class awsAccount1644463885917 implements MigrationInterface {
 
     public async up(queryRunner: QueryRunner): Promise<void> {
         await queryRunner.query(`CREATE TABLE "aws_account" ("id" SERIAL NOT NULL, "access_key_id" character varying NOT NULL, "secret_access_key" character varying NOT NULL, "region" character varying NOT NULL, CONSTRAINT "PK_9ad897024d8c36c541d7fe84084" PRIMARY KEY ("id"))`);
+        await queryRunner.query(`
+            create or replace procedure iasql_schedule_apply()
+            language plpgsql
+            as $$
+            begin
+                PERFORM graphile_worker.add_job('scheduleApply');
+                raise notice 'Scheduled apply';
+            end;
+            $$;
+        `);
+        await queryRunner.query(`
+            create or replace procedure iasql_schedule_sync()
+            language plpgsql
+            as $$
+            begin
+                PERFORM graphile_worker.add_job('scheduleSync');
```

Comment:
how does the bd know the existence of `graphile_worker.*`?

Reply:
it doesn't, we guarantee the schema will be there by starting the scheduler (which does it's own migrations) before we run our own migrations https://github.com/iasql/iasql-engine/pull/521/files#diff-c8f7acf4155d692e9663c0ae07f0ab80ceb39ab0c6d7e15d9a2e987aa1a04ac9R55

Reply:
oh ok, so the 'graphile-worker' run method creates the schema and the procedure add_job

Reply:
yes!